src/BullaFrendLendV2.sol

```diff
@@ -499,6 +544,13 @@ contract BullaFrendLendV2 is BullaClaimControllerBase, ERC165, IBullaFrendLendV2
             revert InvalidCallback();
         }
```

Comment:
Can probably remove validate callback configuration because it's not whitelisted